site/app/templates/grading/electronic/GradingPanelHeader.twig

```diff
@@ -51,7 +51,7 @@
             {{ self.renderPanelPositionSelector('submission_browser') }}
         </span>
         {% if isStudentInfoPanel %}
-            <span class="grade-panel" id="student_info_btn">
+            <span class="grade-panel" id="student_info_btn" data-testid="student_info_btn">
```

Comment:
we should be consistent with the selectors, and use hyphens instead of underscores.

Reply:
yeah i will fixed it, i think i read it in opposite yeah , thanks for pointing me